docs/distributed-cluster-deployment-guide/01-distributed-cluster-deployment-guide.md

```diff
@@ -0,0 +1,24 @@
+# Distributed Consistency Protocol for EventMesh
```

Comment:
The Distributed Consistency Protocol is applicable only to protocols akin to KRaft/JRaft and the like. Reverse Proxy does not fall under the purview of the Distributed Consistency Protocol.

Reply:
I will fix the title into 'Distributed Consistency Protocol or Reverse proxy'

Reply:
You can split them into two documents, or use a more general title, such as "Cluster Design."